eZ/Publish/Core/Repository/Values/ObjectState/ObjectState.php

```diff
@@ -26,16 +23,33 @@
  */
 class ObjectState extends APIObjectState
 {
-    use MultiLanguageTrait;
-    use MultiLanguageNameTrait;
-    use MultiLanguageDescriptionTrait;
```

Comment:
> what about using a (internal documented) constant that can be set to the fallback language propert, main (defualt), inital (versionInfo, but not applicable here) and default in this case

Meaning e.g. 
`const DEFAULT_LANGUAGE_PROPERTY = 'defaultLanguageCode';`
with usage as:
`$this->names[$this->{static::DEFAULT_LANGUAGE_PROPERTY}]`
? :)

Could be, it's PHP after all ;-)
I've created some unit tests as this logic seems to get more complicated and if you're fine with it I'll include them in this PR.